net/dcwifi/dcwapd/Makefile

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright (C) 2019 EWSI
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+# For LINUX_DIR
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=dcwapd
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=1
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=COPYING
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/ewsi/$(PKG_NAME)/tar.gz/v$(PKG_VERSION)?
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_HASH:=750a08abccd88d9aeda942307f76ce5711181c06f9f3e8fded5cb5ce42bac323
+PKG_MAINTAINER:=Carey Sonsino <careys@edgewaterwireless.com>
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_SOURCE_SUBDIR)
+# Note - we use the static directory, but we patch to use shared libs
+BUILD_VERSION=linuxjsonstatic
+PKG_BUILD_BIN_DIR=$(PKG_BUILD_DIR)/dcwapd.$(BUILD_VERSION)
+
+PKG_FIXUP:=autoreconf
+
+include $(INCLUDE_DIR)/uclibc++.mk
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/dcwapd
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=Routing and Redirection
+  TITLE:=Dual-Channel WiFi AP daemon
+  URL:=https://www.edgewaterwireless.com
+  DEPENDS:= $(CXX_DEPENDS) +kmod-macremapper +libdcwsocket +libdcwproto +mrmctl +libuci
+  PKG_BUILD_DEPENDS:= libdcwsocket libdcwproto mrmctl
+endef
+
+define Package/dcwapd/description
+Implementation of the Dual-Channel WiFi AP daemon
+endef
+
+export LINUX_ROOT=$(LINUX_DIR)
+export CROSS_COMPILE=$(TOOLCHAIN_DIR)/bin/$(TARGET_CROSS)
+export SUBDIRS=$(PKG_BUILD_DIR)
+
+CONFIGURE_ARGS += \
+	--enable-platform=$(BUILD_VERSION) \
+	--enable-shared
+
+define Build/Compile
```

Comment:
get rid of this section. Replace it with:

```
TARGET_CXXFLAGS += -std=c++98 -DRAPIDJSON_HAS_CXX11_RVALUE_REFS=0"
TARGET_LDFLAGS += -ldcwproto -ldcwsocket -lmrmfilterparser -luci
```

If you really need to specify the others as well, use MAKE_FLAGS +=